packages/block-library/src/cover/editor.scss

```diff
@@ -65,6 +67,10 @@
 		left: 0;
 	}
 
+	// When image being uploaded.
+	&.is-transient img {
+		opacity: 0.3;
+	}
```

Comment:
I think the `&.is-transient::before` styles already handle this, so there's no need for image opacity.